src/main/webapp/i18n/de/faq.json

```diff
@@ -4,12 +4,19 @@
             "home": {
                 "title": "FAQ",
                 "createLabel": "FAQ erstellen",
+                "proposeLabel": "FAQ vorschlagen",
+                "accept": "FAQ akzeptieren",
+                "reject": "FAQ ablehnen",
                 "filterLabel": "Filter",
                 "createOrEditLabel": "FAQ erstellen oder bearbeiten"
             },
-            "created": "Das FAQ wurde erfolgreich erstellt",
-            "updated": "Das FAQ wurde erfolgreich aktualisiert",
-            "deleted": "Das FAQ wurde erfolgreich gelöscht",
+            "created": "Die FAQ mit der ID {{ id }} wurde erfolgreich erstellt",
+            "updated": "Die FAQ mit der ID {{ id }} wurde erfolgreich aktualisiert",
+            "proposed": "Die FAQ mit der ID {{ id }} wurde erfolgreich vorgeschlagen",
```

Comment:
Does the id really help an instructor?

Reply:
I think this is personal preference. I prefer to get feedback about what i edited instead of an generic message. Otherwise, you might be confused if you accidently clicked on the wrong FAQ / you can double check the faq in the overview easier if you know the id

Reply:
I think the title would be more helpful here 😅

Reply:
added title